src/libs/ReportUtils.js

```diff
@@ -417,12 +433,49 @@ function isPolicyExpenseChatAdmin(report, policies) {
     return policyRole === CONST.POLICY.ROLE.ADMIN;
 }
 
+/**
+ * Returns true if report has a parent and is therefore a Thread.
+ *
+ * @param {Object} report
+ * @returns {Boolean}
+ */
+function isThread(report) {
+    if (!report || !report.parentReportID || !report.parentReportActionID) {
+        return false;
+    }
+    return true;
+}
+
+/**
+ * Returns true if reportAction has a child.
+ *
+ * @param {Object} reportAction
+ * @returns {Boolean}
+ */
+function isThreadParent(reportAction) {
+    if (!reportAction || !reportAction.childReportID || reportAction.childReportID === '0') {
```

Comment:
similar comment here, how about,
```
return _.has(report, 'childReportID');
```
is `0` saved in onyx at some point for childReportID?